src/System/System.php

```diff
@@ -513,8 +516,21 @@ public static function getIOUsage($duration = 1): array
 
         // Compute Delta
         foreach ($diskStat as $key => $disk) {
-            $stats[$key]['read'] = (((intval($diskStat2[$key][5]) - intval($disk[5])) * 512) / 1048576);
-            $stats[$key]['write'] = (((intval($diskStat2[$key][9]) - intval($disk[9])) * 512) / 1048576);
+            $read1 = $diskStat2[$key][5];
```

Comment:
let's make this read2 as it is from second disk stats
```suggestion
            $read2 = $diskStat2[$key][5];
```